google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1460,6 +1463,7 @@ public static Builder newBuilder() {
    * @return a [@code Blob} with complete information
    * @throws StorageException upon failure
    */
+  @Deprecated
```

Comment:
Now I'm wondering is it possible to undeprecate something in the future? It seems like an essential method and we will eventually want to undeprecate it in the future (for example if we manage to implement it reliably with mark()/reset() calls)? Or, I know that we talked about it, but now I think maybe we should not mark this method as deprecated as we do not provide a non-deprecated alternative to this method (the byte[] version is not enough).